pkg/tests/ossm/route_prevent_additional_ingress_test.go

```diff
@@ -0,0 +1,73 @@
+package ossm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+	"github.com/maistra/maistra-test-tool/pkg/util/version"
+)
+
+func TestRoutePreventAdditionalIngress(t *testing.T) {
+	NewTest(t).Id("T48").Groups(Full).MaxVersion(version.SMCP_2_5).Run(func(t TestHelper) {
+
+		ER := "extra-routes"
```

Comment:
This should be more explicit, for example: `extraRoutesNS`. With this when you are reading will be more clear for what will be used